examples/idx_bd_cache/main.go

```diff
@@ -15,59 +15,112 @@
 package main
 
 import (
+	"log"
+	"sync"
+
+	"github.com/ligato/cn-infra/agent"
+	"github.com/ligato/cn-infra/datasync"
+	"github.com/ligato/cn-infra/datasync/kvdbsync"
+	"github.com/ligato/cn-infra/db/keyval/etcd"
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/cn-infra/servicelabel"
 	"github.com/ligato/cn-infra/utils/safeclose"
 	"github.com/ligato/vpp-agent/plugins/vpp"
 	"github.com/ligato/vpp-agent/plugins/vpp/l2plugin/l2idx"
-	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/vpp-agent/plugins/vpp/model/l2"
 )
 
+const agent1, agent2 = "agent1", "agent2"
+
 // Start Agent plugins selected for this example.
 func main() {
-	// Init close channel to stop the example.
-	//exampleFinished := make(chan struct{}, 1)
-	//
-	//// Start Agent with VPP Flavor and ExampleFlavor.
-	//flavor := vppFlavor.Flavor{}
-	//exampleFlavor := ExampleFlavor{
-	//	IdxBdCacheExample: ExamplePlugin{closeChannel: &exampleFinished},
-	//	Flavor:            &flavor, // inject VPP flavor
-	//}
-	//agent := core.NewAgent(core.Inject(&flavor, &exampleFlavor))
-	//
-	//core.EventLoopWithInterrupt(agent, exampleFinished)
-
-	// todo use new flavors && options
+	// Channel used to close the example
+	exampleClosed := make(chan struct{})
```

Comment:
Let's initialize all of these directly when creating ExamplePlugin, using the same name. So we are consistent across all examples.